src/index.js

```diff
@@ -264,6 +283,16 @@ const main = async () => {
   // Write countries total with deaths etc
   writeTempJSON("dsi-local-acquisition", dsiSourceOfInfectionParsed);
 
+  if (intlVaccinations) {
+    writeTempCSV("intl-vaccinations", intlVaccinations);
```

Comment:
I found that using csv instead of json cut the size of the file at least by 4 (2mb vs 400kb) so if your json files are getting huge might be worth looking into that!

Reply:
Thanks, yeah I've thought of that in the past using CSV because it's more size efficient. The only problem is that CSV files don't seem to get gzipped by the server when we fetch them and JSON files do, so we ended up actually transferring more data over the wire. I made a thing a while back for a different project that downloaded 7zip CSV files and unzipped them on the fly and processed them, but in the end it just was easier to use JSON files, especially if a lot of the data is repeated strings, they seem to be shrunk down quite a bit during gzip.

Would be good to find a solution to this though. Maybe we can contact IT and see if they can add csv files to the files that the server automatically gzips.

Reply:
Ah ok, it's not a massive difference between gzipped json and the csv now so I will leave this for now.